### ISSUE TITLE
Build error overlay color problem

### ISSUE THREAD

**Issue by derekpa** [spywhere/Javatar#74]
When an error is displayed over the code after a build the background of the message message is the same as the page - here you can see the close button and the text, which is confusing.

[![Screen Shot 2017-03-11 at 11.54.05.png](https://s18.postimg.org/as3hlrl21/Screen_Shot_2017_03_11_at_11_54_05.png)](https://postimg.org/image/78hjvyic5/)

**Comment by spywhere** [comment]
I don't get it. Could you rephrase your issue again?

**Comment by spywhere** [comment]
Close issue due to inactive from the reporter.

**Comment by derekpa** [comment]
Hi, sorry. I thought the image I attached would explain the problem. The "Error cannot find symbol" message is very hard to notice as its background is transparent/same colour as edit window background.

Cheers.

**Comment by spywhere** [comment]
Could you try on different color scheme? It's seem to me that the color scheme palette is too similar as the syntax itself doesn't contains any hardcoded color at all.
